.github/workflows/client_host.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         target: ["native", "asterisc"]
-        name: ["OP Sepolia (Holocene) - Block #22012816"]
+        name: ["OP Sepolia (Holocene) - Block #26215604"]
```

Comment:
unrelated bug fix? would be beneficial if this can be committed in a separate pr to reduce the diff in this pr as much as possible. are there any other parts of the code that can be scoped out to smaller prs and merged independently?

Reply:
Not unrelatred. `alloy-evm` accesses more state than our legacy block executor, which changed the contents of the trie witness. This update just pulls in a new test block that was crafted with the updated setup for a complete witness.

Reply:
hm I see, so it's not possible to do this part first or shrink scope in any other way?

Reply:
I can extend the stack upwards, but ultimately yes, without this CI will fail on the PR closest to trunk.